cranelift/codegen/src/isa/x64/inst/emit.rs

```diff
@@ -1078,7 +1078,16 @@ pub(crate) fn emit(inst: &Inst, sink: &mut MachBuffer<Inst>) {
                 }
             }
         }
+        Inst::XMM_Mov_R_M { op, src, addr } => {
+            let rex = RexFlags::clear_w();
+            let (prefix, opcode) = match op {
+                SseOpcode::Movd => (LegacyPrefix::_66, 0x0F7E),
```

Comment:
Out of curiosity, is `movd` better than `movsd` in this case?

Reply:
I was first implementing for movd (set in lower.rs) because this was a 32 bit move, else I'd prefer movsd over movq since movsd is intended for fp.

Reply:
Right regarding 32 vs 64 bits; then my question is, is movd better than movss? movss can be encoded with F3 0F 11, so there's no difference in size; and we do use movss for reg to reg moves, and if I'm understanding correctly for f64 we'll use movsd for both RR and RM. I am not asking to change, but trying to understand; the only argument in favor of movss might be consistency with f64 here.

Reply:
Actually, when looking at the output of llvm and gcc, both use movss in this case. Also Intel's optimization guide seems to suggest there might be an extra overhead in using movd for FP registers? (3.5.2.3 Bypass between Execution Domains)
So could we change it to use movss instead, please? (and ditto for loads)